apps/owmweather/lib.js

```diff
@@ -43,7 +62,13 @@ exports.pull = function(completionCallback) {
     "location": "London"
   };
   let settings = require("Storage").readJSON("owmweather.json", 1);
-  let uri = "https://api.openweathermap.org/data/3.0/onecall?lat=" + location.lat.toFixed(2) + "&lon=" + location.lon.toFixed(2) + "&exclude=minutely,hourly,daily,alerts&appid=" + settings.apikey;
+  let uri;
+  if (!!settings.useOneCall) {
```

Comment:
The linter is not happy about the !! here. I think you can just remove this since the `if` can handle all truthy/falsy values.